src/Extend/Settings.php

```diff
@@ -26,11 +26,12 @@ class Settings implements ExtenderInterface
      * @param string $attributeName: The attribute name to be used in the ForumSerializer attributes array.
      * @param string $key: The key of the setting.
      * @param string|callable|null $callback: Optional callback to modify the value before serialization.
+     * @param mixed $default: Optional default serialized value.
```

Comment:
I think it'd be important to mention (and test) that the default value would still be passed through the callback.

Reply:
sure,
the test I added actually already makes sure it passes through the callback, but I guess I'll add another test where there's simply no callback as well.